libraries/chain/db_maint.cpp

```diff
@@ -1225,7 +1244,7 @@ void database::perform_chain_maintenance(const signed_block& next_block, const g
             uint64_t num_committee_voting_stake; // number of committee members
             voting_stake[2] = ( pob_activated ? 0 : stats.total_core_in_orders.value )
                   + (stake_account.cashback_vb.valid() ? (*stake_account.cashback_vb)(d).balance.amount.value: 0)
-                  + stats.core_in_balance.value;
+                  + ( hf2262_passed ? 0 : stats.core_in_balance.value );
```

Comment:
Just FYI: Removing liquid BTS from voting power will strip ref UI users of their voting since you can only stake BTS with CLI.

Questions:
 - Does mobile support staking?
 -  Do you have someone in mind that could update UI? Or does committee have funding for it?
 - cashback voting power is kept intentionally?
 - Liquid tickets means someone is actively seeking a say in governance, I would find it acceptable to give them 1x voting power

Reply:
> 
>     * Liquid tickets means someone is actively seeking a say in governance, I would find it acceptable to give them 1x voting power

If giving liquid tickets 1x voting power, then nothing is changed, the cashback voting power will still be kept intentionally like beos and cn-vote now doing.
Otherwise, cn-vote or beos still can get the cashback voting power from encouraging and seducing their supporter to lock the bts after this update, ok, cn-vote has done this.

I think the root problem is the defect of DPOS governance, the holders didn't need the sustained investment and spending, so they didn't have the enough motivation to govern the chain, especially some holders were zero cost holders.

On this chain, only the witness need sustained investment and spending to maintain the chain like miner of pow.

Reply:
> Removing liquid BTS from voting power will strip ref UI users of their voting since you can only stake BTS with CLI.

Note:  Uptick also supports creating and updating of voting tickets, in latest release.  (Admittedly that is still a cli solution.... but still friendlier than and an alternative to `cli_wallet`.)

Re: liquid tickets.  "liquid" isn't a ticket target type.  One doesn't create "liquid" tickets.  Rather it is a state that a ticket is in while it's still charging towards a lock target.  (Although if the funds in the ticket are votable *while* it is in the charging/"liquid" state, then one could game the system by initiating a lock ticket and cancelling it prior to the `next_auto_update_time`, which, If I'm understanding correctly, would give you a few days of 1x voting power without actually committing you to the lock period.)

Reply:
@sschiessl-bcp wrote:
> Just FYI: Removing liquid BTS from voting power will strip ref UI users of their voting since you can only stake BTS with CLI.

This is not true because the mobile client supports staking already.

> cashback voting power is kept intentionally?

Thanks for noticing this. I overlooked it. Will fix.

>  I would find it acceptable to give them 1x voting power

It can be gamed as @christophersanborn said:
> one could game the system by initiating a lock ticket and cancelling it

Reply:
@shulthz changes to witnesses will be considered in next releases. Thanks for your ideas anyway.